test/broadcast.jl

```diff
@@ -993,3 +993,6 @@ end
 
     @test Cyclotomic() .* [2, 3] == [[1, 2], [1, 2]]
 end
+
+# dottable test
+@test Broadcast.__dot__(:($(^)(t, 2))) == :($(^).(t, 2))
```

Comment:
What's the "user facing" change this allows one to do? Something with `@.`?

Reply:
Nothing user facing is changed. In ModelingToolkit, we interpolate functions into expressions to resolve modules, so we need this overload to use `__dot__ `.